websites/B/BloomHost/presence.ts

```diff
@@ -0,0 +1,111 @@
+const presence = new Presence({
+		clientId: "873728851052752896"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	{ hostname, pathname } = document.location;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "bloom_logo",
+		startTimestamp: browsingTimestamp
+	};
+	if (hostname === "bloom.host") {
+		presenceData.smallImageKey = "globe-solid";
+		presenceData.details = "Viewing the main website";
+		if (pathname.includes("minecraft")) {
+			presenceData.details = "Viewing the main website";
+			presenceData.state = "Reading: Minecraft Plans";
+		} else if (pathname.includes("vps")) {
+			presenceData.details = "Viewing the main website";
+			presenceData.state = "Reading: VPS Plans";
+		} else if (pathname.includes("duck-panel")) {
+			presenceData.details = "Viewing the main website";
+			presenceData.state = "Reading: About the Duck Panel";
+		} else if (pathname.includes("about-bloom")) {
+			presenceData.details = "Viewing the main website";
+			presenceData.state = "Reading: About Bloom";
+		}
+	}
+	if (hostname === "docs.bloom.host") {
```

Comment:
these should all be else ifs

Reply:
I had it as else ifs but the linter/auto checker was throwing errors about it so I removed it. The code still works

Reply:
No it didn't if you did it properly, please fix this otherwise you're sacrificing performance for no good reason

Reply:
What is a better way of doing it then?

Reply:
https://github.com/PreMiD/Presences/pull/6094/commits/d0d8dcfdebeffd9db10d10850788fbccdd332b74

Reply:
https://github.com/PreMiD/Presences/blob/723adbca4d6a8dc8412b39dd1fb87f70fbcaf28b/websites/B/BloomHost/presence.ts

Reply:
it's as simple as making all of these else ifs

Reply:
I did (check the link) and it threw errors

Reply:
Simply just run prettier with --write

Reply:
Still waiting on this

Reply:
I did that and I ran prettier and it still gives a linting error

Reply:
Then you did it wrong. If you need help use the Discord server. I'm not approving this while these if statements are not made into a case or chain of else ifs

Reply:
The code works but linting is telling me to use switch statements